envparse.py

```diff
@@ -114,6 +114,22 @@ def __call__(self, var, default=NOTSET, cast=None, subcast=None,
         return value
 
     @classmethod
```

Comment:
This probably doesn't need to be a separate function. Since it's not likely to be used elsewhere, just adding a dict to `read_env` would be simpler.